Header header
uint32 landmark_count
uint32 descriptor_len
float64[] position_x
float64[] position_y
float64[] position_signature
float64[] descriptor_mat